### PR TITLE
Added HelmToolInstallerV1 task

### PR DESCRIPTION
**Pull request by Anumita**
without the kubectl tool installer

### CODE REVIEW

File: Tasks/HelmInstallerV1/src/utils.ts [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+import tl = require('vsts-task-lib/task');
+import path = require('path');
+import fs = require('fs');
+import * as toolLib from 'vsts-task-tool-lib/tool';
+import * as utils from './utils';
+import * as os from "os";
+import * as util from "util";
+import { WebRequest, sendRequest } from 'utility-common/restutilities';
+
+const uuidV4 = require('uuid/v4');
```
Comment (bansalaseem) [inline Tasks/HelmInstallerV1/src/utils.ts]: Should not we be using the common code? Why clone the code?

File: Tasks/HelmInstallerV1/Strings/resources.resjson/en-US/resources.resjson [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "loc.friendlyName": "Helm tool installer",
+  "loc.helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?linkid=851275)",
+  "loc.description": "Install Helm on agent machine.",
+  "loc.instanceNameFormat": "Install Helm $(helmVersion)",
+  "loc.input.label.helmVersion": "Helm Version Spec",
+  "loc.input.help.helmVersion": "Specify the version of Helm to install",
+  "loc.messages.DownloadStableVersionFailed": "Failed to read latest kubectl version from stable.txt. From URL %s. Using default stable version %s.",
```
Comment (bansalaseem) [inline Tasks/HelmInstallerV1/Strings/resources.resjson/en-US/resources.resjson]: Should we remove kubectl related messages?

File: Tasks/HelmInstallerV1/src/utils.ts (hunk 1 above)
Comment (Anumita) [reply Tasks/HelmInstallerV1/src/utils.ts]: used common code

File: Tasks/HelmInstallerV1/Strings/resources.resjson/en-US/resources.resjson (hunk 2 above)
Comment (Anumita) [reply Tasks/HelmInstallerV1/Strings/resources.resjson/en-US/resources.resjson]: removed messages

**Pull request merged**